src/pages/Explore/TraceExplorationPage.tsx

```diff
@@ -1,13 +1,23 @@
 import React, { useEffect, useState } from 'react';
+import z from 'zod';
+
 import { newTracesExploration } from '../../utils/utils';
 import { TraceExploration } from './TraceExploration';
 import { DATASOURCE_LS_KEY } from '../../utils/shared';
 import { reportAppInteraction, USER_EVENTS_ACTIONS, USER_EVENTS_PAGES } from '../../utils/analytics';
 import { UrlSyncContextProvider } from '@grafana/scenes';
+import { AdHocVariableFilter } from '@grafana/data';
+
+// @ts-ignore new API that is not yet in stable release
+import { useSidecar_EXPERIMENTAL } from '@grafana/runtime';
 
 export const TraceExplorationPage = () => {
+  // We are calling this conditionally, but it will depend on grafana version and should not change in runtime so we
+  // can ignore the hook rule here
+  const sidecarContext = useSidecar_EXPERIMENTAL?.() ?? {};
+
   const initialDs = localStorage.getItem(DATASOURCE_LS_KEY) || '';
-  const [exploration] = useState(newTracesExploration(initialDs));
+  const [exploration] = useState(newTracesExploration(initialDs, getInitialFilters(sidecarContext.initialContext)));
```

Comment:
So this is the kind of feedback I hope for after this is in Ops. The thing is the persistence was added because the Investigations app which makes sense to be kept "alive" as you can add multiple things to it from multiple places. Right now not sure what exactly is the cases where that is not wanted. Maybe we can have it somehow configurable but not sure what exactly the options should be right now.

Reply:
I think this can go like this for testing internally, though it would be nice to make the "X" button more visible somehow or maybe sticky for now? I'm worried some user may be not sure how to get rid of this. Also it may be confusing as it may show up on any tab after it's open, right?

Reply:
I made some changes so the width of the header with the X button is fixed so it never goes out of sight. Later will probably try to design it a bit better. For example I assume it needs some sort of additional action, like "maximize" or "open in separate tab" etc

Reply:
> Also it may be confusing as it may show up on any tab after it's open, right?

Yes the state is persisted. Again not sure what is the best approach here, at some point I will make it configurable as part of also making the URL work properly, so that explore apps will just be persisted in the URL and work sort of normally and having a non URL but persisted in local storage state as some sort of opt in for specific apps.